crowbar_framework/spec/models/api/upgrade_spec.rb

```diff
@@ -178,6 +178,12 @@
         with("state:crowbar_upgrade AND (roles:database-server OR roles:rabbitmq-server)").
         and_return([NodeObject.find_node_by_name("testing.crowbar.com")])
       )
+      allow_any_instance_of(NodeObject).to receive(:run_ssh_cmd).and_return(
```

Comment:
Oh, so I guess this is what you were referring to in the other commit.  Maybe you split the changes into separate commits (if so, thank you for that!) but forgot to split the commit message too? This commit is missing an explanation.